### PR TITLE
Add a FAQ section with the first common issue

### PR DESCRIPTION
**Pull request by kewisch**
As requested, here is a FAQ section with the issue I solved during the my last office hours. Wording changes welcome.

### CODE REVIEW

File: source/faq.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.. _faq:
+
+
+FAQ
+===
+
+WebExtensions experiements have many similarities to tranditional add-ons,
```
Comment (rpl) [inline source/faq.rst]: typo: experiements => Experiments

**Review by rpl — COMMENTED** [review]
Thanks @kewisch!
Follows a small typo and a proposal for an additional related question (and related possible answer).

File: source/faq.rst (hunk 1 above)
Comment (Rob--W) [reply source/faq.rst]: s/tranditional/traditional/

**Comment by kewisch** [comment]
I've fixed the typos, thanks for catching those! @rpl I can't see the proposal for an additional question though, what am I missing?

**Comment by rpl** [comment]
@kewisch ouch, I'm sorry, the additional QA was part of the previous comment but it got lost somehow when I submitted the comment to github :-(
 
It is related to the `chrome.manifest` (and it completes with the intro of the FAQ section where we say that experiments 'have many similarities to traditional add-ons' by point out some of the differences), something like:

```
Can I include a chrome.manifest in an experiment?
-------------------------------------------------

The experiments do not support the 'chrome.manifest', and it will be ignored if included in the experiments add-on source dir. Look at 'How do I get an URI reference to the files in my experiment?' if you need to get an URI which points to assets packaged with the experiment (e.g. an image, an additional jsm file, a framescript etc.).
```

**Comment by rpl** [comment]
Thanks @kewisch!

New FAQ section in the WebExtensions Experiments docs! \o/

**Pull request merged**